app/src/main/res/layout/views_tema_codech1.xml

```diff
@@ -0,0 +1,59 @@
+<?xml version="1.0" encoding="utf-8"?>
+<RelativeLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:layout_width="match_parent" android:layout_height="match_parent"
+    android:background="@color/lightYellow">
+
+    <ImageView
+        android:id="@+id/imageViewRight"
+        android:layout_width="130dp"
+        android:layout_height="141dp"
+        app:srcCompat="@drawable/android_2"
+        android:layout_alignParentRight="true"/>
+
+    <ImageView
+        android:id="@+id/imageViewLeft"
+        android:layout_width="187dp"
+        android:layout_height="137dp"
+        app:srcCompat="@drawable/android_1" />
+
+    <FrameLayout
+
+        android:layout_width="match_parent"
+            android:layout_height="match_parent"
+        android:layout_below="@id/imageViewLeft"
+        android:background="@color/lime" >
+
+        <TextView
+            android:layout_width="355dp"
```

Comment:
Format the code with CTRL + ALT + L and avoid using hardcoded values for both dimensions width and height, especially if they are TextViews